.travis.yml

```diff
@@ -18,9 +18,11 @@ install:
     - export DEV_LIB_PATH=dev-lib
     - if [ ! -e "$DEV_LIB_PATH" ] && [ -L .travis.yml ]; then export DEV_LIB_PATH=$( dirname $( readlink .travis.yml ) ); fi
     - if [ ! -e "$DEV_LIB_PATH" ]; then git clone https://github.com/xwp/wp-dev-lib.git $DEV_LIB_PATH; fi
+    - npm install
```

Comment:
I believe `npm install` is already being run by `travis.install.sh`, per https://github.com/xwp/wp-dev-lib/blob/d478bb3bc2837f8a3a5019aebf8d6bc8b9f53c38/check-diff.sh#L377-L380